test/helpers/record_helper_test.rb

```diff
@@ -117,4 +117,30 @@ class RecordHelperTest < ActionView::TestCase
     assert_equal(true, scan?)
     ENV['SCAN_EXCLUSIONS'] = stored_env
   end
+
```

Comment:
For completeness, this should include a test with no note and multiple summary holdings.